### PR TITLE
Merging agent based tracking changes to main.

### PR DESCRIPTION
**Pull request by dharmez**

### CODE REVIEW

**Review by dharmez — COMMENTED** [review]
Reviewed all changes.

File: components/processing/form_parser/deployment/versions.tf [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = ">= 5.23.0"
+    }
+  }
+
+  provider_meta "google" {
+    module_name = "cloud-solutions/eks-docai"
```
Comment (kingman) [inline components/processing/form_parser/deployment/versions.tf:26]: you need to add a version number in the end in order for it to be picked up by the tracking, for instance 
`cloud-solutions/eks-docai-v1.2`

File: components/processing/form_parser/src/load_data_in_bigquery.py [hunk 2]
```diff
@@ -28,8 +29,10 @@ def load_rows_into_bigquery(rows_to_insert, table_id):
         rows_to_insert: list of rows with each row as json string,
         table_id: bigquery table id
     """
+  USER_AGENT = "cloud-solutions/eks-docai"
```
Comment (kingman) [inline components/processing/form_parser/src/load_data_in_bigquery.py:32]: add version number.

File: components/processing/form_parser/src/main.py [hunk 3]
```diff
@@ -52,10 +53,12 @@ def batch_process_documents(
         gcs_output_uri: GCS directory to store the out json files,
         gcs_input_prefix: GCS directory to store input files to be processed
     """
+  USER_AGENT = "cloud-solutions/eks-docai"
```
Comment (kingman) [inline components/processing/form_parser/src/main.py:56]: add version number

File: components/processing/form_parser/src/load_data_in_bigquery.py (hunk 2 above)
Comment (dharmez) [reply components/processing/form_parser/src/load_data_in_bigquery.py:32]: @anuradha-bajpai-google - Can you make the suggested changes and submit a PR against version1.2?

File: components/processing/form_parser/src/main.py (hunk 3 above)
Comment (dharmez) [reply components/processing/form_parser/src/main.py:56]: @anuradha-bajpai-google - Can you make the suggested changes and submit a PR against version1.2?

File: components/processing/form_parser/deployment/versions.tf (hunk 1 above)
Comment (dharmez) [reply components/processing/form_parser/deployment/versions.tf:26]: @anuradha-bajpai-google - Can you make the suggested changes and submit a PR against version1.2?

**Pull request closed without merging**